src/main/java/net/pms/encoders/MEncoderVideo.java

```diff
@@ -1302,22 +1302,24 @@ public ProcessWrapper launchTranscode(
 						mpeg2Options = mpeg2Options.substring(mpeg2Options.indexOf("/*"));
 					}
 
-					// Determine a good quality setting based on video attributes
-					if (mpeg2Options.contains("Automatic")) {
-						mpeg2Options = "keyint=5:vqscale=1:vqmin=2:vqmax=3";
-
-						// It has been reported that non-PS3 renderers prefer keyint 5 but prefer it for PS3 because it lowers the average bitrate
-						if (params.getMediaRenderer().isPS3()) {
-							mpeg2Options = "keyint=25:vqscale=1:vqmin=2:vqmax=3";
-						}
+					// when the automatic bandwidth is used than use the proper automatic MPEG2 setting
+					if (configuration.isAutomaticMaximumBitrate()) {
+						mpeg2Options = params.getMediaRenderer().getAutomaticVideoQuality();
+					}
 
-						if (mpeg2Options.contains("Wireless") || maximumBitrate < 70) {
```

Comment:
This was in fact the bug. It never checks the `Wireless` because the `mpeg2Options` was set to `"keyint=5:vqscale=1:vqmin=2:vqmax=3"` at the line 1307

Reply:
These limits are old and honestly quite arbitrary. I don't think there ever was a WiFi standard that was 30 Mbps. The biggest "flaw" in the logic IMO is forgetting that the bandwidth is shared between all devices on the network, and forgetting that bandwidth often varies a lot over time on wireless connections. Since bandwidth will vary, it's impossible to actually achieve the "average" on a constant basis. The peaks will be above, and the bottoms will be below, so if you try to send something, you must make sure that you don't surpass the lowest available bandwidth you will experience. Receiver buffering will help to smooth this out, but we don't control how much renderers buffer so we can't really count on that.

So, the actual bandwidth used must be quite a bit lower than what you "measure" if you want to achieve non-interrupted playback. Guessing the delta between the minimum and the average bandwidth is no easy feat, and my guess is that the failure to handle/understand these things is what led to the whole "half bitrate" thing in the first place.

Reply:
I agree that it is outdated now since modern Wi-Fi can handle very high bitrates. At the time, there was a clear speed difference between Ethernet and Wi-Fi, but in everyday terms now with good equipment there doesn't need to be.

So "wireless vs wired" is a concept we might not need anymore - we should probably just make sure our automatic speed measuring good. However the one difference between wired and wireless is that wired is far more stable, as wireless can drop low temporarily for a number of reasons (maybe a new phone joined, or someone's computer starts doing updates in the background, etc.).

The best tools constantly alter bitrate because of this inconsistency, by having the server offer several different bitrates and the client can choose the right one based on how well it is keeping up with buffering, but we are really limited by DLNA in this regard. We can be smarter with our web interface but probably the best thing to do via DLNA is to measure the network speed directly before attempting to play a video, instead of only doing it at startup.

Just some quick thoughts.